osier/models/model.py

```diff
@@ -0,0 +1,80 @@
+import pandas as pd
+import numpy as np
+from unyt import unyt_array, hr, MW, kW, GW
+import itertools as it
+from osier import Technology
+from osier.technology import _validate_quantity, _validate_unit
+from osier.utils import synchronize_units
+import warnings
+import logging
+
+_freq_opts = {'D': 'day',
+              'H': 'hour',
+              'S': 'second',
+              'T': 'minute'}
+
+
+class OsierModel():
```

Comment:
This class has a lot of shared code with `DispatchModel`. Will `DispatchModel` be refactored to inherit from `OsierModel`?

Reply:
Yes, that is the dream. I realized there was going to be significant overlap in these two classes but not enough overlap to warrant directly inheriting from `DispatchModel`.